example_test.go

```diff
@@ -23,13 +23,19 @@ func ExampleNewWatcher() {
 	go func() {
 		for {
 			select {
-			case event := <-watcher.Events:
+			case event, ok := <-watcher.Events:
 				log.Println("event:", event)
 				if event.Op&fsnotify.Write == fsnotify.Write {
 					log.Println("modified file:", event.Name)
 				}
-			case err := <-watcher.Errors:
+				if !ok {
```

Comment:
Check the `ok` before reading from the `event`. Essentially, this needs to happen before line 27.

Reply:
Right :)
Fixed.